modules/cookbooks/pages/snowflake_ingestion.adoc

```diff
@@ -0,0 +1,278 @@
+-- IT WILL BE SOMETHING LIKE organization_name-account_name
+-- e.g. ykmxgak-wyb52636
+WITH HOSTLIST AS 
+(SELECT * FROM TABLE(FLATTEN(INPUT => PARSE_JSON(SYSTEM$allowlist()))))
+SELECT REPLACE(VALUE:host,'.snowflakecomputing.com','') AS ACCOUNT_IDENTIFIER
+FROM HOSTLIST
+WHERE VALUE:type = 'SNOWFLAKE_DEPLOYMENT_REGIONLESS';
+----
+
+=== Create a RSA key pair
+
+Next up we will need to create an https://docs.snowflake.com/en/user-guide/key-pair-auth[RSA key pair^] using `openssl`
+to authenticate Redpanda Connect to Snowflake. You will be prompted to give an encryption password, remember this phrase,
+you will need it later.
+
+[source,bash]
+----
+openssl genrsa 2048 | openssl pkcs8 -topk8 -inform PEM -passout pass:Testing123 -out rsa_key.p8
+----
+
+Next we will create a public key by running following commands. You will be prompted to type in the phrase you used in the above step.
+
+[source,bash]
+----
+openssl rsa -in rsa_key.p8 -pubout -passout pass:Testing123 -out rsa_key.pub
+----
+
+To register our public key in Snowflake, we'll need to remove the public key delimiters and output only the base64 encoded portion of the
+PEM file. You can print that with the following bash command:
+
+[source,bash]
+----
+cat rsa_key.pub | sed -e '1d' -e '$d' | tr -d '\n'
+----
+
+Now going back to our Snowflake worksheet, we'll enter the output of the above command in the following SQL command and execute.
+
+[source,sql]
+----
+use role accountadmin;
+alter user streaming_user set rsa_public_key='< PubKeyWithoutDelimiters >';
+----
+
+=== Create a schema using `streaming_user`
+
+Now logout of Snowflake, sign back in as the default user `streaming_user` we just created with the associated password (default: Test1234567).
+Run the following SQL commands in a worksheet to create a schema (e.g. `STREAMING_SCHEMA`) in the default database (e.g. `STREAMING_DB`):
+
+[source,sql]
+----
+SET DB = 'STREAMING_DB';
+SET SCHEMA = 'STREAMING_SCHEMA';
+USE IDENTIFIER($DB);
+CREATE OR REPLACE SCHEMA IDENTIFIER($SCHEMA);
+----
+
+== Create a pipeline from Redpanda to Snowflake
+
+ifndef::env-cloud[]
+
+With all of our setup out of the way, we're now ready to create our pipeline. We'll run our pipeline locally
+and inject the xref:configuration:secrets.adoc[secrets] using environment variables. First we need to create
+a `connect.yaml` file the following contents.
+
+[source,yaml]
+----
+input:
+  # Read data from our local `demo_topic`
+  kafka_franz:
+    seed_brokers: ["localhost:9092"]
+    topics: ["demo_topic"]
+    consumer_group: "redpanda_connect_to_snowflake"
+    tls: {enabled: true}
+    checkpoint_limit: 4096
+    sasl:
+      - mechanism: SCRAM-SHA-256
+        username: ingestion_user
+        password: ${REDPANDA_PASS}
+    # Define our batching policy. For this cookbook, we'll create small batches
+    # but in a production environment we'll want to create as large of files as
+    # possible.
```

Comment:
```suggestion
    # Define our batching policy. For this cookbook, we create small batches,
    # but a production environment could create files as large as
    # possible.
```

Reply:
I think it should read `but a production environment should create files as large as`